src/app/tabs/analysis-tab.js

```diff
@@ -1,12 +1,32 @@
 var yo = require('yo-yo')
-var css = require('./styles/analysis-tab-styles')
+var csjs = require('csjs-inject')
+var remixLib = require('remix-lib')
 
-function analysisTab (api = {}, events = {}, opts = {}) {
-  var el = yo`
-    <div class="${css.analysisTabView} "id="staticanalysisView">
-    </div>
-  `
-  return { render () { return el } }
-}
+var EventManager = remixLib.EventManager
 
-module.exports = analysisTab
+module.exports = class AnalysisTab {
+  constructor (opts = { api: {}, events: {} }) {
+    const self = this
+    self.event = new EventManager()
+    self._api = opts.api
+    self._events = opts.events
+    self._view = { el: null }
+    self.data = {}
+    self._components = {}
+  }
+  render () {
+    const self = this
+    if (self._view.el) return self._view.el
+    self._view.el = yo`
+      <div class="${css.analysisTabView} "id="staticanalysisView"></div>`
+    return self._view.el
+  }
+}
+const css = csjs`
```

Comment:
seems some CSS has been lost `staticanalysisView`, `infoBox`, `textBox`

Reply:
Yes, but i think i removed it because i checked that it wasn't in use or didn't make a difference, because another class was overwriting it or it simply put stuff on that was already inherited or a default. ...so basically instead of just copy/pasting, i was trying to refactor the styling too to get rid of redundant styling attributes